openid-connect-server/src/test/java/org/mitre/oauth2/service/impl/TestDefaultOAuth2AuthorizationCodeService

```diff
@@ -0,0 +1,53 @@
+package org.mitre.oauth2.service.impl;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.when;
+import org.junit.Test;
+import org.mitre.oauth2.model.AuthenticationHolderEntity;
+import org.mitre.oauth2.model.AuthorizationCodeEntity;
+import org.mitre.oauth2.repository.AuthorizationCodeRepository;
+import org.mitre.oauth2.repository.impl.JpaAuthenticationHolderRepository;
+import org.mitre.oauth2.repository.impl.JpaAuthorizationCodeRepository;
+import org.mockito.Matchers;
+import org.mockito.Mockito;
+import org.springframework.security.oauth2.provider.OAuth2Authentication;
+
+public class TestDefaultOAuth2AuthorizationCodeService {
+
+	@Test
+	public void testConsumeAuthorizationCode()
```

Comment:
the formatting of this function is off, please format like other code in the project before submission